code/modules/power/supermatter/supermatter.dm

```diff
@@ -178,7 +195,14 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/bullet_energy = 2
 	///How much hallucination should we produce per unit of power?
 	var/hallucination_power = 0.1
-
+	///The efficiency of the antinoblium interaction.
+	var/antinoblium_efficiency = 0.0
+	///Delta from antinoblium_efficiency and antinoblium composition.
+	var/antinoblium_efficiency_delta = 0.0
+	///Inverse of 1 - antinoblium_efficiency.
```

Comment:
What does this mean

Reply:
antinoblium_efficiency is a variable that converges to antinoblium composition. It will have strong effects that change a lot of things about the SM the closer it approaches to 100%.

Reply:
Documenting just the value isn't helpful. you can do both, but why is more useful then what.

Like, what you've written here is better then just `Inverse of 1 - antinoblium_efficiency.`

Reply:
Storing just the inverse is a bit weird, but I see why you're doing it

Reply:
I'll add better documentation when I get on my PC.